docs/source/public_api/problems.rst

```diff
@@ -8,20 +8,19 @@ Definitions and collection of problems solved by probabilistic numerical methods
     :no-main-docstr:
 
 probnum.problems.zoo
-********************
+--------------------
 
 Subpackage offering implementations of standard example problems or convenient
 interfaces to benchmark problems for probabilistic numerical methods. These test
 problems are meant for rapid experimentation and prototyping.
 
 probnum.problems.zoo.diffeq
----------------------------
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
the docs failed, because the subsection was not consistent somehow. This way it works, but if there is a better fix, let me know.